tests/js/setup-globals.js

```diff
@@ -1,6 +1,6 @@
 // Set up `wp.*` aliases.  Doing this because any tests importing wp stuff will likely run into this.
 global.wp = {};
-
+require( '@wordpress/data' );
```

Comment:
Required so that when mocking `@wordpress/data` it doesn't complain about missing `AsyncModeProvider`